homeassistant/components/surepetcare/__init__.py

```diff
@@ -139,17 +139,22 @@ def __init__(self, hass: HomeAssistant, surepy: Surepy) -> None:
         """Initialize the Sure Petcare object."""
         self.hass = hass
         self.surepy = surepy
-        self.states = {}
+        self.states: dict[int, Any] = {}
 
     async def async_update(self, _: Any = None) -> None:
         """Get the latest data from Sure Petcare."""
 
+        _LOGGER.debug("🐾  updating sure petcare data... ")
+
         try:
-            self.states = await self.surepy.get_entities()
+            self.states = await self.surepy.get_entities(refresh=True)
```

Comment:
You could consider reverting this, and add it in a separate pr. Then we can include it in 2021.8

Reply:
Tested this locally. And each update will now trigger two requests to petcare servers.
Is this needed? 
https://github.com/benleb/surepy/blob/1d684016dc6d31747b73f0bc4c7bd520258549ac/surepy/__init__.py#L363
You do not do anything with the value returned from get_actions ?

Reply:
@benleb Any update on this?

Reply:
sorry... I will try to fix this asap!